File: Habitica/src/main/java/com/habitrpg/android/habitica/data/local/FAQLocalRepository.java
package com.habitrpg.android.habitica.data.local;

import com.habitrpg.android.habitica.models.FAQArticle;

import java.util.List;

import io.realm.RealmResults;
import rx.Observable;

public interface FAQLocalRepository extends ContentLocalRepository {

    Observable<RealmResults<FAQArticle>> getArticles();
}

File: Habitica/src/main/java/com/habitrpg/android/habitica/data/local/FAQLocalRepository.kt
package com.habitrpg.android.habitica.data.local

import com.habitrpg.android.habitica.models.FAQArticle
import kotlinx.coroutines.flow.Flow

interface FAQLocalRepository : ContentLocalRepository {
    fun getArticle(position: Int): Flow<FAQArticle>

    val articles: Flow<List<FAQArticle>>
}
